src/styles/components/clock.scss

```diff
@@ -0,0 +1,49 @@
+.clock {
+  padding: 400px;
+
+  &__center {
+    position: relative;
+
+    width: 30px;
+    height: 30px;
+
+    border: 2px solid rgb(115, 115, 255);
+    border-radius: 50%;
+    background-color: #fff;
+    box-shadow: 0 0 0 2px #fff;
```

Comment:
if you use hash for colors `#fff` in the code you should use it in all places, don't combine rgb and hash in the styles